src/dotnet/Chat/Chat.cs

```diff
@@ -31,10 +31,11 @@ public sealed record Chat(
     [DataMember] public string Title { get; init; } = "";
     [DataMember] public Moment CreatedAt { get; init; }
     [DataMember] public bool IsPublic { get; init; }
-    [DataMember] public string Picture { get; init; } = "";
+    [DataMember] public MediaId MediaId { get; init; } = MediaId.None;
```

Comment:
Isn't MediaId a struct hence None by default? unless default is not None

Reply:
Done